resources/test-data.md

```diff
@@ -225,26 +226,26 @@ table below. There will be no user dialog at the mobile phone when testing
 Vipps.
 
 {:.table .table-striped}
-| Mobile number | Error message                                                   |
+| Mobile number | Error message |
 | :------------ | :-------------------------------------------------------------- |
-| 99999991      | Vipps internal error                                            |
-| 99999992      | Request Validation error message on paticular request parameter |
-| 99999993      | Transaction Id already exists in vipps                          |
-| 99999994      | PSPID not enrolled in vipps                                     |
-| 99999995      | Invalid payment model type                                      |
-| 99999996      | User Vipps App version not supported                            |
-| 99999997      | User not Registered with Vipps                                  |
-| 99999998      | Merchant not available or active                                |
+| 99999991 | Vipps internal error |
```

Comment:
The spaces removed here was there to make the table look neat and well-aligned in plain text inside an editor (or here on GitHub). Markdown is not only a source for HTML-conversion, it's also meant to be readable in and of itself, as plain text. 😃